Travis CI fails with Node.js 13.3.0+

The latest version of Node.js (v13.4.0, v13.5.0 or v13.6.0) make the Travis CI fail, for example, https://travis-ci.com/oeway/ImJoy/builds/143990637

It complains:
```
$ npm prune
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.8 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.8: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! invalid bin entry for package @babel/parser@7.1.6. key=parser, value=bin/babel-parser.js
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2020-01-11T07_40_15_888Z-debug.log
The command "npm prune" failed and exited with 1 during .
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,10 +4,8 @@ addons:
   firefox: latest
 language: node_js
 node_js:
-  - node
-cache:
-  directories:
-    - node_modules
+  - "13.3.0"
+cache: npm
 notifications:
   email: false
 before_script:
@@ -15,6 +13,7 @@ before_script:
   # - "sh -e /etc/init.d/xvfb start"
   # - sleep 3 # give xvfb some time to start
   - cd ./web
+  - npm prune
 script:
   - export NODE_OPTIONS="--max-old-space-size=4096"
   - npm run check-format

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,7 @@ addons:
   firefox: latest
 language: node_js
 node_js:
-  - "13.5.0"
+  - "13.4.0"
 cache: npm
 notifications:
   email: false

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,7 @@ addons:
   firefox: latest
 language: node_js
 node_js:
-  - node
+  - "13.3.0"
 cache:
   directories:
     - node_modules

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,7 @@ addons:
   firefox: latest
 language: node_js
 node_js:
-  - "13.3.0"
+  - node
 cache:
   directories:
     - node_modules
@@ -15,7 +15,6 @@ before_script:
   # - "sh -e /etc/init.d/xvfb start"
   # - sleep 3 # give xvfb some time to start
   - cd ./web
-  - npm prune
 script:
   - export NODE_OPTIONS="--max-old-space-size=4096"
   - npm run check-format

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,7 @@ addons:
   firefox: latest
 language: node_js
 node_js:
-  - "13.3.0"
+  - "13.5.0"
 cache: npm
 notifications:
   email: false

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,7 @@ addons:
   firefox: latest
 language: node_js
 node_js:
-  - "13.4.0"
+  - "13.3.0"
 cache: npm
 notifications:
   email: false

